command/pipe.go

```diff
@@ -0,0 +1,355 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"mime"
+	"os"
+	"path/filepath"
+
+	"github.com/urfave/cli/v2"
+
+	errorpkg "github.com/peak/s5cmd/v2/error"
+	"github.com/peak/s5cmd/v2/log"
+	"github.com/peak/s5cmd/v2/log/stat"
+	"github.com/peak/s5cmd/v2/storage"
+	"github.com/peak/s5cmd/v2/storage/url"
+)
+
+var pipeHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] destination
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	01. Upload stdin output to a bucket
+		 > echo "content" | s5cmd {{.HelpName}} s3://bucket/prefix/object.gz
```

Comment:
let's make example better by doing exactly what it says:

`echo "object content" | gzip | ./s5cmd pipe s3://bucket/prefix/object.gz`